crates/oxc_transformer/Cargo.toml

```diff
@@ -26,11 +26,18 @@ oxc_diagnostics = { workspace = true }
 oxc_syntax      = { workspace = true, features = ["to_js_string"] }
 oxc_traverse    = { workspace = true }
 
-rustc-hash = { workspace = true }
-indexmap   = { workspace = true }
-serde      = { workspace = true, features = ["derive"] }
-serde_json = { workspace = true }
-ropey      = { workspace = true }
+dashmap         = { workspace = true }
+indexmap        = { workspace = true }
+serde           = { workspace = true, features = ["derive"] }
+serde_json      = { workspace = true }
+ropey           = { workspace = true }
+rustc-hash      = { workspace = true }
+once_cell       = { workspace = true }
+st-map          = { version = "0.2.3" }
+semver          = { version = "1.0.23" }
+anyhow          = { version = "1.0.86" }
+tracing         = { version = "0.1.40" }
+browserslist-rs = { version = "0.15.0", features = ["wasm_bindgen"] }
```

Comment:
do we need wasm bindgen?

Reply:
Yes, oxc-wasm needs

Reply:
Need to remove `st-map`, `semver`, `tracing`

Reply:
All of these are removed!